pkg/client/connector/auth/license.go

```diff
@@ -76,9 +76,10 @@ func (l *loginExecutor) GetLicense(ctx context.Context, id string) (string, erro
 		return "", err
 	} else if license, err := getLicenseJWT(ctx, l.env, tokenInfo.AccessToken, id); err != nil {
 		return "", err
-	} else if license == "" {
-		return "", fmt.Errorf("No licenses found")
 	} else {
+		if license == "" {
```

Comment:
That was what I had but the linter didn't like it  and wanted to replace it with a switch which I thought would be even more ugly, let me see if I can find a way to make it work.

Reply:
The linter is also wrong. It cannot be replaced with a switch because some of the if's contain assignments. Switch cases cannot have assignments. So, never mind.

Reply:
That's weird that that's what the linter said--the linter is usually smart enough to not suggest a switch when there are assignments.